automation/README.md

```diff
@@ -0,0 +1,226 @@
+# FHE Toolkit - Hyper Protect Deployment Automation
+
+IBM Cloud Hyper Protect Virtual Servers (HPVS) make it possible to sign and deploy applications built on the FHE Toolkit into a product-grade hardened environment and runtime.  However, the manual process for deploying the FHE Toolkit and its' derivatives into an HPVS environment are complex and time-consuming.  Automating this process can help to close the technical gaps that prevent enterprise adaptation of production FHE solutions.
```

Comment:
In the first sentence where it says "Hyper Protect Virtual Servers (HPVS)" I would make that text link to the Hyper Protect Virtual Server info page for people who want to know more. The link is here for reference: https://www.ibm.com/cloud/hyper-protect-virtual-servers